src/components/Sum.jsx

```diff
@@ -1 +1,14 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum(props) {
+  return (
+    <p>
+      {`Sum of ${props.a} and ${props.b} is ${props.a + props.b}`}
+    </p>
+  );
+}
+
+Sum.defaultProps = {
```

Comment:
good! also, you can add default value as you did it in functions

`export function Sum({ a = 0, b = 0}) {...}`